pgdedupe/cli.py

```diff
@@ -3,24 +3,22 @@
 """
 Based on: https://github.com/datamade/dedupe-examples/tree/master/pgsql_big_dedupe_example
 """
-import os
-import csv
-import tempfile
 import time
 import logging
 
-import random
-import numpy
-
-import json
-import yaml
 import psycopg2 as psy
 import psycopg2.extras
 
 import click
-import dedupe
 
-from . import exact_matches
+from .utils import load_config, filename_friendly_hash, create_model_definition
+from .run import process_options,\
```

Comment:
I feel like the name of this module can be changed to something else than `run`. It's not a big deal tho.